packages/compass-indexes/src/components/indexes/indexes.tsx

```diff
@@ -87,7 +102,7 @@ export const Indexes: React.FunctionComponent<IndexesProps> = ({
         isWritable={isWritable}
         isReadonlyView={isReadonlyView}
         readOnly={readOnly}
-        errorMessage={error}
+        errorMessage={regularError || searchError}
```

Comment:
There are now two error vars: one for regular indexes and one for search indexes. But the toolbar shows the error. I'm passing both as one var which is not ideal. What if indexes broke and then you browsed to search indexes and that broke as well? Maybe a bit of an extreme edge case, but really messy nonetheless.

Reply:
We disable the create index button if there's an error. Which means if one of them errors you can't create either of them. That will probably byte us eventually.